### ISSUE TITLE
Add CI/CD with GitHub action

### ISSUE THREAD

**Issue by malcolmSansen** [Sansen-Shimizu/sakura-boot#8]
Add automation for check, testing (unit, integration functional), documentation for the CI.
Add automation for deployment to maven central for the CD.

- [ ] CI
- [ ] CD

**Comment by vr-varad** [comment]
hey I would like to work on this issue if its open for contribution

**Comment by malcolmSansen** [comment]
Yes, the project is open for contributions, and I’d really appreciate any help.
I think I’ll open a discussion for this issue so we can share about it.
